include/merlin_hashtable.cuh

```diff
@@ -958,149 +963,154 @@ class HashTable {
    * @brief Save keys, vectors, metas in table to file or files.
    *
    * @param file A BaseKVFile object defined the file format on host filesystem.
-   * @param buffer_size The size of buffer used for saving in bytes.
+   * @param batch_size Amount of Key-Value/Meta pairs to save at a time. Larger
+   * values *can* lead to higher performance. However, the memory consumption
+   * also increases.
    * @param stream The CUDA stream used to execute the operation.
    *
    * @return Number of KV pairs saved to file.
    */
   size_type save(BaseKVFile<K, V, M, DIM>* file,
-                 const size_type buffer_size = 1048576,
+                 const size_t batch_size = 1L * 1024 * 1024,
```

Comment:
Batch is easier for developers of the table. Buffer size is for who use 'save', since in many case we only have small mount of memory left, while having lots of tables. Consider every table's DIM is trivial.

Reply:
@rhdong Which variant do you prefer?

Reply:
> @rhdong Which variant do you prefer?

I prefer the original naming :)
const size_type buffer_size = 1048576,